.github/workflows/publish-generator.yml

```diff
@@ -130,6 +133,8 @@ jobs:
           context: ./generators/python
           file: ./generators/python/pydantic/Dockerfile
           platforms: linux/amd64,linux/arm64
+          cache-from: type=gha
```

Comment:
Meganit: do we want to specify a cache-from here?

Also should we do this for every docker push step?

Reply:
oh woops didnt realize there were so many more of them. ill add them below too.

> Meganit: do we want to specify a cache-from here?
are you suggesting putting it somewhere else? i think this is the best spot for it.